notebooks/roc_plot_generator.py

```diff
@@ -534,7 +534,8 @@ def get_acdc_runs(
             try:
                 if latest_file is None:
                     raise wandb.CommError("a")
-                with latest_file.download(ROOT / run.name, replace=True, exist_ok=True) as f:
+                # replace=False because these files are never modified. Save them in a unique location, ROOT/run.id
```

Comment:
Hmm, isn't this problematic if we e.g generated data that is bad data, such as the old perfect greaterthan data?

Reply:
Technically before yes because `run.name` is not unique, but now that we switched to `run.id`, it's not a problem because the runs are different.

That is, the old perfect greaterthan data will remain on disk but be stored somewhere else and won't be used by the roc plot generation.

Reply:
Isn't `run.id` just the `wandb` run ID?

If so, the problem with bad data was on our side, i.e after getting the correct `wandb` data, we counted edges with incorrect logic and then saved things to disk - so I don't see how this is a fix?

Reply:
Hmm. Then we should fix it elsewhere -- these are just the raw files downloaded from wandb. What was exactly the problem? There was the one that created AUC=1 but you fixed that.

Reply:
Oh sorry @rhaps0dy , I'm being an idiot - totally overlooked that we're downloading HF files - for some reasons I thought we were saving our processed files here